Makefile

```diff
@@ -27,6 +27,17 @@ fast-install:
 	cp epidose/device/shutdown_epidose.sh \
 	        epidose/device/upload_seeds_d.sh \
 		epidose/device/update_filter_d.sh \
+		epidose/device/update_software_d.sh \
 		epidose/device/util.sh \
 		epidose/device/wps_scanner_d.sh $(OPT)/bin/
 	supervisorctl restart epidose:*
+
+# Install after software update are obtained from the production branch
+remote-update:
```

Comment:
Didn't we agree that we wanted the update process to be flexible and script based? How can we upgrade the kernel, or the charging microcontroller firmware with this process?